<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录 - XIAOJUSURVEY</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #rgb(68, 105, 120);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 28px;
font-weight: bold;
color: #2c3e50;
margin: 20px;</string>
     </property>
     <property name="text">
      <string>琢  亚  咨  询</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="loginGroupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font-weight: bold;
    border: 1px solid #ddd;
    border-radius: 5px;
    margin-top: 10px;
    padding-top: 15px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
}</string>
     </property>
     <property name="title">
      <string>用户登录</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="usernameEdit">
        <property name="styleSheet">
         <string notr="true">padding: 8px;
border: 1px solid #ddd;
border-radius: 4px;</string>
        </property>
        <property name="placeholderText">
         <string>请输入用户名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="styleSheet">
         <string notr="true">padding: 8px;
border: 1px solid #ddd;
border-radius: 4px;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="captchaLabel">
        <property name="text">
         <string>验证码:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="captchaEdit">
        <property name="styleSheet">
         <string notr="true">padding: 8px;
border: 1px solid #ddd;
border-radius: 4px;</string>
        </property>
        <property name="placeholderText">
         <string>请输入验证码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="captchaImageLabel">
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ddd;
background-color: #eee;
padding: 5px;
min-width: 80px;</string>
        </property>
        <property name="text">
         <string>验证码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:disabled {
    background-color: #bdc3c7;
}</string>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registerButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    border: none;
    padding: 10px;
    border-radius: 4px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #27ae60;
}

QPushButton:disabled {
    background-color: #bdc3c7;
}</string>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="styleSheet">
      <string notr="true">color: #7f8c8d;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>准备就绪</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
